mlir/test/mlir-miopen-lib/applicability/no_ultra_large_kernels.mlir

```diff
@@ -0,0 +1,9 @@
+// XFAIL: *
+// RUN: mlir-miopen-lib-test --option=bin --args "--operation conv2d \
+// RUN: --arch amdgcn-amd-amdhsa:gfx908 --num_cu 64 --in_type fp32 \
```

Comment:
Hmm do we need all that lines to start with `RUN`?

Reply:
Yeah, that's how not having one massively long line (which is a pain for me because screen reader support = no word wrap) works in lit